presto-main/src/main/java/com/facebook/presto/event/QueryMonitor.java

```diff
@@ -273,6 +279,34 @@ private QueryContext createQueryContext(SessionRepresentation session, Optional<
         return Optional.empty();
     }
 
+    private Optional<String> createJsonQueryPlan(QueryInfo queryInfo)
+    {
+        try {
+            if (queryInfo.getOutputStage().isPresent()) {
```

Comment:
I'm confused by this block of code.. Can't we simply use the serialized string directly?